salt/serializers/keyvalue.py

```diff
@@ -0,0 +1,119 @@
+"""
+    salt.serializers.keyvalue
+    ~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    Implements keyvalue serializer which can be used for serializing or
+    deserializing any file which defines keys and values separated by a common
+    set of characters, such environment files, which are in "KEY=value" format.
+
+    Options:
+
+    :param line_ending:
+        String representation of LF or CRLF to be used for serialization to a
+        file. Defaults to ``\\r\\n`` on Windows and ``\\n`` on other operating
+        systems.
+
+    :param quoting:
+        Boolean flag to determine if values should be quoted (``True``) during
+        serialization or dequoted (``False``) during deserialization.
+
+    :param separator:
+        String representing the character(s) used when concatenating or reading
```

Comment:
Presumably you mean the separator between the key and the value?

It also doesn't say what the default is.